UnityProject/Assets/Scripts/HealthV2/Living/BodyPartMutations.cs

```diff
@@ -284,10 +288,135 @@ public void OnDestroy()
 	}
 
 
-	public void ChangeToSpecies(PlayerHealthData PlayerHealthData, GameObject BodyPart)
+	public void ChangeToSpecies(GameObject BodyPart, CharacterSheet characterSheet = null)
 	{
 		if (this.TryGetComponent<Brain>(out var brain)) return; //Make it a little bit harder to remove from a round
-		StartCoroutine(ProcessChangeToSpecies(PlayerHealthData, BodyPart));
+		if (characterSheet != null)
+		{
+			PerfomChangeToSpecies(BodyPart, characterSheet);
+		}
+		else
+			StartCoroutine(ProcessChangeToSpecies(BodyPart));
+	}
+
+	private void SettingUpSubOrgans(BodyPart SpawnedBodypart, ItemStorage bodyPartExampleStorage, bool HasOpenProcedure)
+	{
+		var usedOrgansInSpawnedPart = new List<GameObject>();
+
+		if (bodyPartExampleStorage.Populater.SlotContents.Count() != 0)
```

Comment:
There might be scenarios where the using both

Reply:
> There might be scenarios where the using both

?????

Reply:
Ok, gonna add them all then